torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -57,6 +57,14 @@ inline Tensor relu6(Tensor& input, const ReLU6Options& options) {
     HardtanhOptions().min_val(0).max_val(6).inplace(options.inplace()));
 }
 
+inline Tensor rrelu(Tensor& input, const RReLUOptions& options, bool training=false) {
```

Comment:
similarly here: `options` can be optional, and we should add a test for the no options case too.